string GEOM_BOX="box"
string GEOM_SPHERE="sphere"
string GEOM_CYLINDER="cylinder"
string GEOM_CAPSULE="capsule"

string geom_type               
geometry_msgs/Vector3 extents  
float64 height                 
float64 radius                 
geometry_msgs/Pose initial_pose
float64 mass                   
std_msgs/ColorRGBA color